frontend/src/components/dashboard/BalanceSummary.vue

```diff
@@ -35,73 +45,86 @@
+				>Settle Up</Button
+			>
 		</template>
 	</Card>
 
 	<ErrorMessage :message="summaryResource.error" />
 
 	<SettlementDialog v-model="showSettleUpDialog" />
 
-	<Dialog :options="{ title: `History with ${selectedFriend.full_name}` }" v-model="showTransactionHistoryDialog">
+	<Dialog
+		:options="{ title: `History with ${selectedFriend.full_name}` }"
+		v-model="showTransactionHistoryDialog"
+	>
 		<template #body-content>
 			<div>
 				<ol class="space-y-3">
-					<li class="text-gray-800 font-medium" v-for="item in transactionHistoryResource.data || []">
-						<div class="flex flex-row space-x-2 items-center" v-if="item.is_settlement">
+					<li
+						class="text-gray-800 font-medium"
+						v-for="item in transactionHistoryResource.data || []"
+					>
+						<div
+							class="flex flex-row space-x-2 items-center"
+							v-if="item.is_settlement"
+						>
 							<FeatherIcon class="w-5 text-green-700" name="dollar-sign" />
 							<div>
-								{{ getDisplayNameForHistory(item.credit_user) }} paid <span class="font-bold">₹{{ item.amount }}</span> to {{ getDisplayNameForHistory(item.debit_user) }}.
+								{{ getDisplayNameForHistory(item.debit_user) }} paid
+								<span class="font-bold">₹{{ item.amount }}</span> to
+								{{ getDisplayNameForHistory(item.credit_user) }}.
 							</div>
 						</div>
 						<div class="flex flex-row space-x-2 items-center" v-else>
 							<FeatherIcon class="w-5 text-amber-700" name="file" />
 							<div>
-								{{ getDisplayNameForHistory(item.debit_user) }} borrowed <span class="font-bold">₹{{ item.amount }}</span> from {{ getDisplayNameForHistory(item.credit_user) }}.
+								{{ getDisplayNameForHistory(item.credit_user) }} borrowed
+								<span class="font-bold">₹{{ item.amount }}</span> from
+								{{ getDisplayNameForHistory(item.debit_user) }}.
 							</div>
 						</div>
 					</li>
 				</ol>
 			</div>
 		</template>
 	</Dialog>
-
-
 </template>
 
 <script setup>
-import { reactive, ref } from "vue";
-import { createResource, LoadingText, Card, Avatar, Dialog, FeatherIcon} from "frappe-ui";
-import SettlementDialog from "../dashboard/SettlementDialog.vue";
-import {sessionUser} from "@/data/session";
+import { reactive, ref } from 'vue'
+import { createResource, LoadingText, Card, Avatar, Dialog, FeatherIcon } from 'frappe-ui'
+import SettlementDialog from '../dashboard/SettlementDialog.vue'
+import { sessionUser } from '@/data/session'
 
-const showSettleUpDialog = ref(false);
-const showTransactionHistoryDialog = ref(false);
+const showSettleUpDialog = ref(false)
+const showTransactionHistoryDialog = ref(false)
 
 const summaryResource = createResource({
-	url: "batwara.api.get_summary_for_session_user",
-	auto: true
+	url: 'batwara.api.get_summary_for_session_user',
+	auto: true,
 })
 
-const selectedFriend = reactive({name: "friend3@email.com", full_name: "Shivam Ghosh"})
+const selectedFriend = reactive({ name: 'friend3@email.com', full_name: 'Shivam Ghosh' })
```

Comment:
We have to get rid of this hard coded object.

Reply:
Ok bro,
Can you explain about the error in checking that has happened

Reply:
Linters are failing